samples/typescript_nodejs/00.empty-bot/package.json

```diff
@@ -18,7 +18,7 @@
         "url": "https://github.com"
     },
     "dependencies": {
-        "botbuilder": "~4.14.0",
+        "botbuilder": "~4.15.0-rc0",
```

Comment:
I would avoid making a dependency on a release candidate. Suggest 4.14.1 if any change.

Reply:
Hi @BruceHaley, we are waiting for the 4.15 release to update all the samples. Meanwhile, we use the RC version to test MSI support.